src/Oscoin/Telemetry/Options.hs

```diff
@@ -0,0 +1,63 @@
+module Oscoin.Telemetry.Options
+    ( Options(..)
+
+    , telemetryOptionsParser
+    , telemetryOptionsOpts
+    , renderTelemetryOptionsOpts
+    )
+where
+
+import           Oscoin.Prelude hiding (option)
+
+import qualified Oscoin.Telemetry.Logging as Log
+
+import qualified Formatting as F
+import           Options.Applicative
+import           System.Console.Option
+
+data Options = Options
+    { optLogSeverity :: Log.Severity
+    , optLogStyle    :: Log.StyleFormatter
+    } deriving (Eq, Show)
+
+telemetryOptionsParser :: Parser Options
+telemetryOptionsParser = Options
+    <$> option auto
+        ( long    "log-level"
+       <> help    "Set the logging verbosity"
+       <> metavar (enumBounded (Proxy @Log.Severity))
+       <> value   Log.Debug
+       <> showDefault
+        )
+    <*> styleParser
+  where
+    styleParser = Log.StyleFormatter
+        <$> switch
+            ( long    "no-log-colours"
+           <> help    "Turn off terminal colors for log output"
+            )
+        <*> option auto
+            ( long    "log-layout"
+           <> help    "Control if and how to layout log messages"
+           <> metavar (enumBounded (Proxy @Log.LayoutFormat))
+           <> value   Log.HumanReadable
+           <> showDefault
+            )
+
+    enumBounded
+        :: forall proxy a. (Show a, Enum a, Bounded a)
+        => proxy a
+        -> String
+    enumBounded _ = intercalate "|" $ map (show @a) [minBound..maxBound]
```

Comment:
This looks like a generally-useful function, I would be in favour of having it at the top level 😉

Reply:
Sure... but ... where?

Reply:
I do wonder if something _like Oscoin.Prelude_ (but perhaps not the prelude directly) might be a good candidate 🤔

Reply:
I don't think this is in any way a prelude-ish function, it applies specifically to `optparse-applicative` `metavar`s, where you'd want to display the available options from an enum. If you can think of more options-related things, that may justify some options utils fns module.